packages/ember-headless-form/src/-private/components/field.gts

```diff
@@ -266,6 +266,19 @@ export default class HeadlessFormFieldComponent<
     return this.value;
   }
 
+  get valueAsStringOrNumber(): string | number | undefined {
+    assert(
+      `Only string values are expected for ${String(
```

Comment:
Should this be:

```suggestion
      `Only string or number values are expected for ${String(
```

Reply:
I actually just overlooked that! Thanks for the suggestion, will apply that!